docs/development_environment.md

```diff
@@ -68,6 +68,8 @@ instead.
     SLACK_API_KEY=xoxb-<we'll get to this soon>
     FRONT_END_HOST=https://penny-<your_name>.herokuapp.com
```

Comment:
maybe add a bullet right at the end of this section to visit `https://penny-<your_name>.herokuapp.com` and see that it worked?